python/samples/concepts/chat_completion/image_input.py

```diff
@@ -0,0 +1,78 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+import asyncio
+import logging
+
+from semantic_kernel import Kernel
+from semantic_kernel.connectors.ai.function_call_behavior import FunctionCallBehavior
+from semantic_kernel.connectors.ai.open_ai import AzureChatCompletion
+from semantic_kernel.contents import ChatHistory, ChatMessageContent, ImageContent, TextContent
+
+logging.basicConfig(level=logging.WARNING)
+
+system_message = """
+You are a image reviewing chat bot. Your name is Mosscap and you have one goal
+critiquing images that are supplied.
+"""
+
+kernel = Kernel()
+
+service_id = "chat-gpt"
+chat_service = AzureChatCompletion(service_id=service_id)
```

Comment:
should we name this sample `azure_image_input` or something similar so we can have other examples with other models in the future?